Source/Core/VideoBackends/Metal/MTLUtil.mm

```diff
@@ -503,6 +497,23 @@ fragment float4 fbfetch_test(float4 in [[color(0), raster_order_group(0)]]) {
 
   for (auto& binding : resource_bindings)
     compiler.add_msl_resource_binding(binding);
+  if (stage == ShaderStage::Pixel)
+  {
+    for (u32 i = 0; i < VideoCommon::MAX_PIXEL_SHADER_SAMPLERS; i++)  // ps/samp0-N
+      compiler.add_msl_resource_binding(
```

Comment:
These loops (here and below) should be braced, since their body travels more than one line.

Reply:
Whoops, they were one-liners before clang-format